packages/patternfly-4/react-core/src/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './beta';
```

Comment:
I'm still a bit confused about why we have a beta location - the goal was to move the components to their final place (with the other components) and just use the documentation to indicate they are beta so we wouldn't need to keep moving them.

Reply:
I see, sorry I missed this! Getting that updated now